hist/histv7/test/fill.cxx

```diff
@@ -5,9 +5,43 @@
 // Test Fill(), FillN(), GetEntries(), GetBinContent(), GetBinUncertainty()
 
 
-// TODO : add tests to check 0 before hand + test uncertainty after another fill
 
 
+// RHist::Fill for 1D
+
+// Test Fill() without weight and GetEntries()
+TEST(HistFillTest, FillCoordEntries) {
+  ROOT::Experimental::RH1F hist({100,0.,1});
+  EXPECT_EQ(0, hist.GetEntries());
+  hist.Fill({0.1111});
+  EXPECT_EQ(1, hist.GetEntries());
+  hist.Fill({0.1111});
+  EXPECT_EQ(2, hist.GetEntries());
+}
+
+// Test Fill() without weight and GetBinContent()
+TEST(HistFillTest, FillCoordContent) {
+  ROOT::Experimental::RH1F hist({100,0.,1});
+  EXPECT_FLOAT_EQ(.0f, hist.GetBinContent({0.1111}));
+  hist.Fill({0.1111});
+  EXPECT_FLOAT_EQ(1.0f, hist.GetBinContent({0.1111}));
+  hist.Fill({0.1111});
+  EXPECT_FLOAT_EQ(1.0f+1.0f, hist.GetBinContent({0.1111}));
+}
+
+// Test Fill() without weight and GetBinUncertainty()
+TEST(HistFillTest, FillCoordUncertainty) {
+  ROOT::Experimental::RH1F hist({100,0.,1});
+  float weight1 = .0f;
```

Comment:
Why a variable for `weight1`? Why not just `EXPECT_FLOAT_EQ(.0f, hist.GetBinUncertainty({0.1111}));

Reply:
I wanted to make sure that `sqrt(.0f*.0f)` was actually equal `.0f` but I guess it may be pointless... I'll remove it!